### ISSUE TITLE
Error with Parse SDK 1.7.2

### ISSUE THREAD

**Issue by logicsb** [parse-community/ParseLiveQuery-iOS-OSX#189]
Needs update the code to conform with new Parse SDK 1.7.2

Error:
ParseLiveQuery/Parse+LiveQuery.swift:14:61: Cannot call value of non-function type 'ParseClientConfiguration?'

Thanks

**Comment by noobs2ninjas** [comment]
Just to verify I am assuming the version of Parse in question is actually 1.17.2? If so the work has already begun to update ParseLiveQuery to the latest dependency version possible.

**Comment by noobs2ninjas** [comment]
I'm going to close this issue due to lack of response. Feel free respond if this is still an issue and we'll address it.

**Comment by drdaz** [comment]
I just started running into this.

Updated my pods, and this started. I'm running the Parse version in the PR to update FB libraries to 5.x.

**Comment by smuravev** [comment]
Hi,

The issue still exist unfortunately.
If I use latest versions of `Parse` & `ParseLiveQuery` frameworks then I always get this error:
```
Error:
ParseLiveQuery/Parse+LiveQuery.swift:14:61: Cannot call value of non-function type 'ParseClientConfiguration?'
```

Here are versions of latest Parse and ParseLiveQuery frameworks what downloaded through Cocoapods:

* Parse (1.17.3)
* ParseLiveQuery (2.6.1)

The code in the `ParseLiveQuery (2.6.1)` > `Pods/ParseLiveQuery/Parse+LiveQuery.swift` is still the same unfortunately:

```
import Parse

extension Parse {
    static func validatedCurrentConfiguration() -> ParseClientConfiguration {
        // !!!!!!!!! LOOK HERE please: Parse.currentConfiguration() is used instead of Parse.currentConfiguration !!!!!!!
        guard let configuration = Parse.currentConfiguration() else {
            preconditionFailure("Parse SDK is not initialized. Call Parse.initializeWithConfiguration() before loading live query client.")
        }
        return configuration
    }
}
```

Can you please let me know when you plan to fix it in your distributed frameworks ?

**Comment by taliwolk** [comment]
Running into the same issue.

This little issue is really a huge issue, since it prevents us from updating ParseLiveQuery,
which prevent us form updating Parse SDK itself,
which prevents us form updating Parse/FacebookUtils,
which prevents us from updating FBSDKLoginKit,
which prevents us from updating FBSDKCoreKit
which prevents us from updating FBAudienceNetwork (which we really need for proper ios 13 support).

In short, it makes us work with so much outdated code :-( 
A fix will be highly appreciated.

**Comment by drdaz** [comment]
@noobs2ninjas Is there any reason why we can't make a quick patch for this issue? 

It's a real problem I think, that it looks like we can fix by deleting 2 characters 😃

**Comment by noobs2ninjas** [comment]
So our current release(2.6.1) supports parse 1.17.0. This error occurs specifically on Parse 1.17.3. We actually have [this pull request](https://github.com/parse-community/ParseLiveQuery-iOS-OSX) that will update all this but Parse is having a build issue with a [pull request]( https://github.com/parse-community/Parse-SDK-iOS-OSX/pull/1460 ) that has build issues with Bolts.

In easy terms. Bolts(a key dependency of Parse) is no longer being updated. That wasn't such a big deal until you start building with Xcode 11 which brought some MAJOR changes involving the build process. As soon as we get Parse updated to reflect those changes everything will go back to normal and it should be much easier to keep things updated. Thanks so much everyone for your patients. I promise once we get over this little bump everything should resume timely updates and improvements.

Also a quick note. Like many of you, I found myself making issues due to lack of consistent maintenance and updates. After, finding out that Parse had lost the key maintainer of LiveQuery I decided to jump into action and more or less take responsibility for PLQ. I'm doing my best to get us back to regular updates like we had about year ago or so. If any of you would like to help I'd appreciate it. If not, give us a week or so and I think we will be all good.

**Comment by noobs2ninjas** [comment]
Unfortunately, its not quite as easy as @drdaz mentioned because we cant simply change the two characters. We have to officially add the dependency of Parse 1.17.3 and update Carthage as well as Starscream. This requires also updating git submodules as well as our Cartfile.lock plus a few build settings. Otherwise I'd have pushed this update a long time ago. Good news is that the pull requests listed in the previous comment has all of that finished. Just waiting on our Parse build to pass.

**Comment by taliwolk** [comment]
@noobs2ninjas Thank you for responding and clarifying the situation and the hold-up. Your efforts are much appreciated!

**Comment by jeffreyjackson** [comment]
Thanks for the help on this!

**Comment by drdaz** [comment]
Should be fixed in the 2.7.0 release.
